servers/zms/schema/updates/update-20230530.sql

```diff
@@ -0,0 +1,12 @@
+CREATE TABLE IF NOT EXISTS `zms_server`.`service_tags` (
+   `service_id` INT UNSIGNED NOT NULL,
+   `key` VARCHAR(64) CHARACTER SET 'utf8' COLLATE 'utf8_bin' NOT NULL,
+    `value` VARCHAR(256) NOT NULL,
+    PRIMARY KEY (`service_id`, `key`, `value`),
+    INDEX `tag_key_val_sec_idx` (`key` ASC, `value` ASC),
+    CONSTRAINT `fk_serivce_tag_service`
+    FOREIGN KEY (`service_id`)
```

Comment:
can we indent the lines 8-11 to be consistent with the definition in the zms_server.sql